napari/components/_tests/test_layer_slicer.py

```diff
@@ -62,6 +63,9 @@ def _get_slice(self, request: FakeSliceRequest) -> FakeSliceResponse:
     def _is_async(self) -> bool:
         return True
 
+    def _slice_dims(self, *args, **kwargs) -> None:
+        return None
```

Comment:
See https://github.com/napari/napari/pull/5299/files#r1017092496

```suggestion
        self.slice_count += 1
```